git-tar/function/ops.go

```diff
@@ -493,3 +493,75 @@ func getShortSHA(sha string) string {
 	}
 	return sha[:7]
 }
+
+func reportStatus(status *sdk.Status, SCM string) {
+	if SCM == "github" {
+		reportGitHubStatus(status)
+	} else {
+		reportGitLabStatus(status)
+		for i := range status.CommitStatuses {
+			delete(status.CommitStatuses, i)
```

Comment:
I think we can avoid the deletions by creating a new status object for the final stack status. Can you try that out?

Reply:
Breaking change. Needs careful testing before update.

Reply:
Amended fix as suggested ^